src/HttpResponse.cpp

```diff
@@ -49,16 +52,12 @@ void HttpResponse::setResponseBody(HttpRequest *request, Server *server)
 	}
 	else
 	{
-		// TODO add redirection from configurated root to the index html
+		// TODO add redirection from configurated root to the index
 		std::ios_base::openmode mode = setContentType(request->getResourcePath());
 		setBody(FileHandler::getFileResource(request, mode));
 	}
 }
 
-HttpResponse::~HttpResponse(void)
```

Comment:
we should always have a deconstructor!

Reply:
Oh shit, this was not on purpose. I'll add it back in, just a minute

Reply:
acutally I have a deconstructor in my http response, line 37

Reply:
I think probably just changed its place, not sure why it happened